deployment/provision_vagrant_vm.sh

```diff
@@ -37,11 +37,10 @@ service redis-server restart
 apt-get -q install -y apache2 apache2-dev libapache2-mod-wsgi-py3
 
 # With docker for mac, root will own the mount point (uid=0). chmod does not touch the host file system in these cases.
-OWNER=$(stat -c %U "$MOUNTPOINT/evap")
-if [ "$OWNER" == "root" ]; then chown -R 1042 "$MOUNTPOINT"; fi
+if [ $(stat -c "%u" "$MOUNTPOINT/evap") == "root" ]; then chown -R 1042 "$MOUNTPOINT"; fi
```

Comment:
For your setup where this doesn't work,
* what does `stat -c "%u" "$MOUNTPOINT/evap"` return here? If it doesn't return "root", maybe we could instead check if the owning-user-id is 0?
* does `chown -R 1042 "$MOUNTPOINT"` work, so that the mounting point is afterwards owned by user id 1042 when viewed from inside the docker container?

Also: What vagrant and docker versions are you using?